x-pack/plugins/security_solution/public/entity_analytics/routes.tsx

```diff
@@ -26,22 +31,88 @@ const EntityAnalyticsTelemetry = () => (
   </PluginTemplateWrapper>
 );
 
-const EntityAnalyticsContainer: React.FC = () => {
+const EntityAnalyticsManagementContainer: React.FC = React.memo(() => {
   return (
     <Switch>
-      <Route path={ENTITY_ANALYTICS_MANAGEMENT_PATH} exact component={EntityAnalyticsTelemetry} />
+      <Route
+        path={ENTITY_ANALYTICS_MANAGEMENT_PATH}
+        exact
+        component={EntityAnalyticsManagementTelemetry}
+      />
       <Route component={NotFoundPage} />
     </Switch>
   );
-};
+});
+EntityAnalyticsManagementContainer.displayName = 'EntityAnalyticsManagementContainer';
 
-const EntityAnalytics = React.memo(EntityAnalyticsContainer);
+const EntityAnalyticsAssetClassificationTelemetry = () => (
+  <PluginTemplateWrapper>
+    <TrackApplicationView viewId={SecurityPageName.entityAnalyticsAssetClassification}>
+      <AssetCriticalityUploadPage />
+      <SpyRoute pageName={SecurityPageName.entityAnalyticsAssetClassification} />
+    </TrackApplicationView>
+  </PluginTemplateWrapper>
+);
 
-const renderEntityAnalyticsRoutes = () => <EntityAnalytics />;
+const EntityAnalyticsAssetClassificationContainer: React.FC = React.memo(() => {
+  return (
+    <Switch>
+      <Route
+        path={ENTITY_ANALYTICS_ASSET_CRITICALITY_PATH}
+        exact
+        component={EntityAnalyticsAssetClassificationTelemetry}
+      />
+      <Route component={NotFoundPage} />
+    </Switch>
+  );
+});
+
+EntityAnalyticsAssetClassificationContainer.displayName =
+  'EntityAnalyticsAssetClassificationContainer';
 
 export const routes = [
   {
     path: ENTITY_ANALYTICS_MANAGEMENT_PATH,
-    render: renderEntityAnalyticsRoutes,
+    component: EntityAnalyticsManagementContainer,
+  },
+  {
+    path: ENTITY_ANALYTICS_ASSET_CRITICALITY_PATH,
+    component: EntityAnalyticsAssetClassificationContainer,
   },
 ];
+
+//  const UsersRoutes = () => (
```

Comment:
Whats all this commented code?

Reply:
Garbage. I think I might have skipped a commit when cherry-picking. 
Deleted!